packages/components/package.json

```diff
@@ -38,6 +38,8 @@
 		"@emotion/styled": "^11.6.0",
 		"@emotion/utils": "^1.0.0",
 		"@floating-ui/react-dom": "1.0.0",
+		"@types/gradient-parser": "0.1.2",
```

Comment:
I'm not sure what `@types/gradient-parser` is needed for 🤔

Reply:
The `gradient-parser` library is used in the `CustomGradientPicker` component, and therefore we've been using the types since the component was refactored to TypeScript.